### PR TITLE
show long circle is T5 but not perfectly normal

### LINKED ISSUE #754: Long circle is T_5 but not T_6
https://topology.pi-base.org/spaces/S000196

Clearly $\omega_1$ is a subspace of the long circle so its not $T_6$
Moreover, any its proper subspaces is a subspace of a LOTS, so normal.

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #754.

Need to justify that perfectly normal is a hereditary property first.

### CODE REVIEW

**Comment by prabau** [comment]
Great idea about adding hereditary in a meta-properties section for T6
(similar to what was done in https://topology.pi-base.org/properties/P000109 for example).

Lots of references available, Engelking, mathse, etc.

**Comment by StevenClontz** [comment]
e.g. https://math.stackexchange.com/questions/1840614/perfectly-normal-is-hereditary

**Comment by Moniker1998** [comment]
Given the following definitions, that perfectly normal property is hereditary would be the easiest to show from "every closed set is a zero-set".
For if $Y\subseteq X$ and $A\subseteq Y$ is closed, then $\overline{A}$ is a zero-set of $X$ so $A = \overline{A}\cap Y$ is a zero-set of $Y$.

**Comment by Moniker1998** [comment]
Or just reference a theorem https://topology.pi-base.org/theorems/T000156

**Comment by prabau** [comment]
I agree with @Moniker1998 that to show that P15 (perfectly normal) is hereditary, it's nearly immediate if we use the characterization of P15 as "closed sets are zero sets".  That is in fact the proof in Engelking 2.1.6.  So the easiest would be to quote that theorem. That way, we don't have to spend a lot of real estate on the page to validate the meta-property.

I don't like the https://math.stackexchange.com/questions/1840614/perfectly-normal-is-hereditary too much as a reference, because the poster is sloppy.  His definition of perfectly normal is not even correct.

**Comment by prabau** [comment]
The arguments for the S196 traits are insufficient I think.

For P15, we should mention that perfectly normal is a hereditary property (without justification).

For P8 (T5), we also need to show the whole space is T4.
Also, can you explain to me why every proper subspace is a LOTS?

**Comment by prabau** [comment]
To show that the long circle is T5, we need:
- show $X$ is T4
- show every proper subspace is T4, it's enough to show every subspace obtained by removing a single point is T5.  That's because it's a LOTS, as you mentioned.  To see this, the argument in https://topology.pi-base.org/spaces/S000196/properties/P000204 gives the idea.

**Comment by yhx-12243** [comment]
I think we can show instead Long circle (S196) is not countably tight (P81), and thus is not perfectly normal automatically.

https://github.com/pi-base/data/pull/990/files#diff-f42ba7d033486c9e0008cb8cb76cb858df0ca916f52c3d1d2609d5ef097b97ce

**Comment by yhx-12243** [comment]
S196|!P81

**Comment by yhx-12243** [comment]
> To show that the long circle is T5, we need:
> 
> * show $X$ is T4
> * show every proper subspace is T4, it's enough to show every subspace obtained by removing a single point is T5.  That's because it's a LOTS, as you mentioned.  To see this, the argument in https://topology.pi-base.org/spaces/S000196/properties/P000204 gives the idea.

For $X$ is $T_4$, we take any two disjoint closed $A, B$. By connectivity there exists $p \in X \setminus (A \cup B)$. Then we use the normality of $X \setminus \{p\}$ to get $A \subseteq U$ and $B \subseteq V$. Then $U \setminus \{p\}$ and $V \setminus \{p\}$ is what we want.

**Comment by prabau** [comment]
@yhx-12243 FYI, if you want to show braces in mathematics formula here, one needs to use double backslashes:
`$\\{x\\}$` giving $\\{x\\}$, and not `$\{x\}$`.

File: spaces/S000196/properties/P000008.md
Comment (pzjp) [inline spaces/S000196/properties/P000008.md:1]: Not every proper subspace is orderable (orderability is not hereditary). But it is GO {P154}.

**Comment by StevenClontz** [comment]
Thanks all for the feedback. Working on this now.

File: properties/P000015.md [hunk 1]
```diff
@@ -20,3 +22,9 @@ in Theorem 1.5.19 of {{zb:0684.54001}} (under the assumption of a {P2} space, bu
 equally valid without this assumption).  See also {{mathse:72138}}.
 
 Defined on page 16 of {{doi:10.1007/978-1-4612-6290-9}} (as perfectly \(T_4\)).
+
+----
+
+#### Meta-properties
+
+- This property is hereditary (see {{mathse:1840621}}).
```
Comment (prabau) [inline properties/P000015.md:30]:
```suggestion
- This property is hereditary (see Theorem 2.1.6 in {{zb:0684.54001}}).
```

File: properties/P000015.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ refs:
     name: General Topology (Engelking, 1989)
   - mathse: 72138
     name: Why are these two definitions of a perfectly normal space equivalent?
+  - mathse: 1840621
+    name: Answer to "Perfectly Normal is hereditary"
```
Comment (prabau) [inline properties/P000015.md:12]:
```suggestion
```

**Comment by prabau** [comment]
S196 P8 (T5 property): This is not enough.

As explained before, it's not because every proper subspace is T5 that the whole space will be.  Also need to show that the whole space is T4.  (Right now, T4 follows from T5, which is circular reasoning.)

Also, the justification that every proper subspace is a GO-space is unsupported.  Need to show why, or refer to some existing mathse post.

**Comment by pzjp** [comment]
It is elementary that after removing arbitrary point we obtain LOTS. And the space cannot be covered by two nonempty closed sets (connectedness), hence they are contained in a LOTS which is T4.
(Also this is a *loop-ordered space* mentioned in [this paper](https://zbmath.org/1458.54023) )

**Comment by prabau** [comment]
> It is elementary that after removing arbitrary point we obtain LOTS. And the space cannot be covered by two nonempty closed sets (connectedness), hence they are contained in a LOTS which is T4. (Also this is a _loop-ordered space_ mentioned in [this paper](https://zbmath.org/1458.54023) )

I am not saying it's difficult.  But something more should be mentioned for P8.

Interesting paper by the way.  I need to find the time to look at it in more detail.

**Comment by StevenClontz** [comment]
Discussion of complete normality is now factored out to https://math.stackexchange.com/questions/5017774

**Comment by prabau** [comment]
Thanks for adding the mathse post.

File: spaces/S000196/properties/P000008.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000196
+property: P000008
+value: true
+refs:
+    mathse: 5017774
+    name: Why is the "long circle" completely normal ($T_5$)?
```
Comment (prabau) [inline spaces/S000196/properties/P000008.md:7]:
```suggestion
refs:
  - mathse: 5017774
    name: Why is the "long circle" completely normal ($T_5$)?
```
fixes build problem

**Review by prabau — APPROVED** [review]

**Pull request merged**